style.css

```diff
@@ -309,3 +320,202 @@ ul {
   font-size: 24px;
   color: var(--white-color);
 }
+
+.navbar-black {
+  display: none;
+}
+
+.partner {
+  display: none;
+}
+
+footer {
+  display: none;
+}
+
+/* Desktop version */
+
+@media only screen and (min-width: 768px) {
+  .navbar {
+    background-color: var(--white-color);
+    height: 58px;
+  }
+
+  .navbar-black {
+    display: inline;
+    width: 100%;
+    height: 24px;
+    background-color: var(--dark-color);
+    flex-direction: column;
+  }
+
+  .navbar-black ul {
+    flex-direction: row;
+    display: flex;
+    justify-content: flex-end;
+    gap: 12px;
+    padding: 4px;
+    padding-right: 25px;
+    margin: 0;
+    background-color: var(--dark-color);
+  }
+
+  .navbar-black li {
+    color: var(--white-color);
+  }
+
+  .menu-list {
+    display: flex;
+    gap: 12px;
+  }
+
+  #menu-icon {
+    display: none;
+  }
+
+  .desktop-nav-item {
+    color: var(--dark-color);
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  h3 {
+    margin-top: 180px;
+    font-weight: 500;
+    font-size: 48px;
+  }
+
+  h2 {
+    font-weight: 700;
+    font-size: 56px;
+  }
+
+  .hello-section {
+    padding-left: 12%;
+    padding-right: 12%;
+  }
```

Comment:
- [ ] You did an awesome job with the transparent design, however, kindly stick to the background nature of the design given 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/227091318-01a8ec1f-52bf-4787-8ebe-e11ad07758df.png)


## Design:
![image](https://user-images.githubusercontent.com/50990925/227091473-559ff4b9-928d-48e8-b603-040c56b57c7d.png)

Reply:
👍👍